.github/kokoro/run-vtr.sh

```diff
@@ -18,6 +19,8 @@ source $SCRIPT_DIR/steps/git.sh
 
 if [ $VTR_TEST == "vtr_reg_strong" ] || [ $VTR_TEST == "odin_reg_strong" ]; then
 	source $SCRIPT_DIR/steps/vtr-min-setup.sh
+elif [ $VTR_TEST == "odin_tech_strong" ]; then
```

Comment:
I have added an appropriate comment. Basically, we need to separate it from the other two tests since Yosys needs to be installed first and the Odin-II tech map will run after that